src/t8_forest/t8_forest.cxx

```diff
@@ -2948,6 +2951,19 @@ t8_forest_set_balance (t8_forest_t forest, const t8_forest_t set_from, int no_re
   }
 }
 
+void
+t8_forest_set_ghost_ext_new (t8_forest_t forest, int do_ghost, t8_forest_ghost_interface_c *ghost_interface)
+{
+  T8_ASSERT (t8_forest_is_initialized (forest));
+  SC_CHECK_ABORT (do_ghost != 0, "do_ghost == 0 in set_ghost_ext_new.\n");
+  SC_CHECK_ABORT (ghost_interface != NULL, "invalides ghost interface in set_ghost_ext_new\n");
```

Comment:
I see your point. But a little question, why should we use `T8_ASSERT` instead of `SC_CHECK_ABORT` ?
If someone sets the combination 1 and NULL, this can not work.

Reply:
> I see your point. But a little question, why should we use `T8_ASSERT` instead of `SC_CHECK_ABORT` ? If someone sets the combination 1 and NULL, this can not work.
> 
> In the function `t8_forest_set_ghost_ext` we also use `SC_CHECK_ABORT`, that was the reason for me to do the same.

We want to use `T8_ASSERT` as often as possible, because it does not affect the performance in release mode. In this case a `SC_CHECK_ABORT` is also okay, because this function is not called that often